src/main/java/softdesign/Map.java

```diff
@@ -5,20 +5,22 @@
 public class Map {
 
 	public static final int WALL = -1;
-  public static int pointsVisted;
+  public static int pointsVisited;
```

Comment:
`pointsVisited` can be `private int` instead of `public static int`.

Reply:
ok